test/verify/check-networking

```diff
@@ -120,7 +120,7 @@ class TestNetworking(MachineCase):
 
         # Configure a manual IP address
         #
-        b.click("tr:contains('IPv4') button")
+        b.click("tr:contains('IPv4') a", True)
```

Comment:
I would put a ```console.log("click visible", sel, el.offsetWidth, el.offsetHeight)``` in ph_click() where we check the offsetWidth and offsetHeight. I've had to tweak this once before, so there's a chance that it doesn't account for certain corner cases.